lib/bucketUtils.js

```diff
@@ -54,10 +53,10 @@ function emptyBucket(aws, bucketName) {
     const contents = resp.Contents;
 
     if (!contents[0]) {
-      return BbPromise.resolve();
+      return Promise.resolve();
     } else {
-      const objects = _.map(contents, function(content) {
-        return _.pick(content, 'Key');
```

Comment:
This actually broke the required format. Fixing this.

Reply:
Kept lodash out and just modified what you had slightly in the final version to 

return {Key: content.Key};

That's the structure the AWS SDK expects so it knows what the S3 object key is. Lodash was doing this for us but it wasn't as transparent which is probably why you missed it.